docs/DeveloperGuide.md

```diff
@@ -264,14 +264,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**System:**
```

Comment:
Should this **System** section be removed since we have specified the system at line 265?